portal-impl/src/com/liferay/portal/service/impl/WorkflowDefinitionLinkLocalServiceImpl.java

```diff
@@ -382,6 +408,41 @@ public WorkflowDefinitionLink updateWorkflowDefinitionLink(
 		return workflowDefinitionLinkPersistence.update(workflowDefinitionLink);
 	}
 
+	@Override
+	public WorkflowDefinitionLink updateWorkflowDefinitionLink(
+			String className, Long groupId,
```

Comment:
Hi @JhossephG , the className should come after the groupId, that is the order on service.xml and the other methods in this class. Please, after updating also check the usages.